build/Jenkinsfile

```diff
@@ -1,52 +1,80 @@
 #!/usr/bin/env groovy
 
-def gitCommit
-node {
-    stage('Checkout') {
-        checkout scm
-        gitCommit = sh(returnStdout: true, script: 'git rev-parse HEAD').trim()
+pipeline {
+    agent {
+        docker {
+            image 'mattermost/mattermost-build-webapp:v5.2.0'
```

Comment:
we will build new docker images for every release? or just we need to upgrade node and your friends?

Reply:
My thinking was that we'd only actually upgrade the base image as we needed to. So 5.3, 5.4 and friends might build on this image, but 5.5 requiring `node:10` would introduce a new such image. I'm using the Mattermost version as a "time stamp" of sorts.